data.sql

```diff
@@ -3,3 +3,6 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) values(1,'Agumon','2020-02-03
+',0,false,10.23),(2,'Gabumon','2018-11-15',2,true,8),(3,'Pikachu','2021-01-07',1,false,15.04),(4,'Devimon','2017-05-12',
```

Comment:
- [ ] Kindly make sure that neutered is true for Agumon as per the requirements.

Reply:
- @sja-thedude  Thank you noticing that , suggested changes made 👍🏽 
![image](https://user-images.githubusercontent.com/94127418/178602454-88814dd5-0eba-44fe-9af9-08278702db42.png)